src/ngraph/frontend/onnx_import/utils/common.hpp

```diff
@@ -86,6 +89,40 @@ namespace ngraph
                 }
             }
 
+            /// \brief Creates a shifted square identity matrix.
```

Comment:
I think it's worth adding some explanation that "shifted" means an offset from the main diagonal.